src/views/Login/Ledger.js

```diff
@@ -164,7 +164,8 @@ export default function Ledger({ className, goHome }) {
         application. If you're running on older firmware, make sure the "browser
         support" option is turned on. To sign transactions, you'll also need to
         enable the "contract data" option. You will be prompted to sign a
-        message. This allows Bridge to operate correctly.
+        message. This allows Bridge to operate correctly. Never sign this hash
+        outside of Bridge.
```

Comment:
> Never sign this hash

Should be "sign this message", right? `Trezor.js` does this correctly.

This warning feels kinda ehh, but there really isn't much more we can do here. Maybe the message should be "urbit bridge exclusive token" or similar to drive this home, and look fishy when prompted for by other sites?

Reply:
Ledgers display a hash when a message is requested to be signed, so I thought consistency with the hardware trumped copy consistency.

Reply:
Fair thinking, but iirc Trezors display (the beginning of) the message itself, so those two aren't consistent even with each other. I think in such cases it's best to err on the side of describing the underlying action accurately? If they have trouble, you want them to search for "ledger sign message", etc etc.